### ISSUE TITLE
width&height configurable

### ISSUE THREAD

**Issue by kongxiaojian123** [vue-gl-fx/vue-gl-fx#2]
If I want to improve the resolution of canvas, I can't do it now. Now I just set it to clientwidth and clientheight

**Comment by Norbz** [comment]
You're right, this should be done quite easily. I'll look into it as soon as I can
